<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>782</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>362</width>
    <height>782</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>362</width>
    <height>782</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>шаблоны</string>
  </property>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>40</y>
     <width>300</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;ШАБЛОНЫ ЙОУ&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="pic_3">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>319</y>
     <width>150</width>
     <height>150</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>150</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../sbln/3_1.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="pic_6">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>497</y>
     <width>150</width>
     <height>150</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>150</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../sbln/5_1.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="pic_4">
   <property name="geometry">
    <rect>
     <x>187</x>
     <y>319</y>
     <width>150</width>
     <height>150</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>150</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../sbln/4_1.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="pic_1">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>141</y>
     <width>150</width>
     <height>150</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>150</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../sbln/1_1.png</pixmap>
   </property>
   <property name="openExternalLinks">
    <bool>false</bool>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByMouse</set>
   </property>
  </widget>
  <widget class="QLabel" name="pic_2">
   <property name="geometry">
    <rect>
     <x>187</x>
     <y>141</y>
     <width>150</width>
     <height>150</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>150</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../sbln/2_1.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="pic_5">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>187</x>
     <y>497</y>
     <width>150</width>
     <height>150</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>150</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../sbln/6_1.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="choose">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>710</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Использовать</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="pic1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>290</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Эта?</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="pic2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>290</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Эта?</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="pic3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>480</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Эта?</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="pic4">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>470</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Эта?</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="pic5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>650</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Эта?</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="pic6">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>660</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Эта?</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
